fvt/testdata/isvcs/isvc-pvc-3.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: isvc-pvc3
+  labels:
+    name: isvc-pvc3
```

Comment:
NIT: Can the label here be removed?

Reply:
I used it to fetch the predictor for debugging purposes since the actual isvc name is different every run

Reply:
removed now